server/helpers/page.js

```diff
@@ -73,8 +73,8 @@ module.exports = {
    */
   injectPageMetadata(page) {
     let meta = [
-      ['title', page.title],
-      ['description', page.description],
+      ['title', '"' + page.title + '"'],
+      ['description', '"' + page.description + '"'],
```

Comment:
Would be a good idea to escape both variables in case they contain " in them as well.